flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -76,7 +80,7 @@ public SqlCreateTable(
 			SqlCharStringLiteral comment) {
 		super(OPERATOR, pos, false, false);
 		this.tableName = requireNonNull(tableName, "Table name is missing");
-		this.columnList = requireNonNull(columnList, "Column list should not be null");
+		this.columnList = columnList;
```

Comment:
Add `requireNonNull` on all the fields should not be null.

Reply:
Done. To distinguish two kind of fields(which can't be missing and which can be missing but can't be null), I add different error messages.